packages/tabs/src/LionTabs.js

```diff
@@ -238,18 +238,19 @@ export class LionTabs extends LitElement {
   /**
    * @param {HTMLButtonElement[]} tabs
    * @param {HTMLButtonElement} currentTab
+   * @param {'right' | 'left'} dir
    * @returns {HTMLButtonElement|undefined}
    * @private
    */
-  __getNextNotDisabledTab(tabs, currentTab) {
-    const notDisabledTabs = tabs.filter(tab => !tab.disabled);
-    const isLastNotDisabledTab =
-      currentTab === notDisabledTabs[0] ||
-      currentTab === notDisabledTabs[notDisabledTabs.length - 1];
-    if (isLastNotDisabledTab) {
-      return notDisabledTabs.find(tab => tab !== currentTab);
+  __getNextNotDisabledTab(tabs, currentTab, dir) {
+    let nextNotDisabledTabs = /** @type {HTMLButtonElement[]} */ ([]);
+    if (dir === 'right') {
+      nextNotDisabledTabs = tabs.filter((tab, i) => !tab.disabled && i > this.selectedIndex);
+      return nextNotDisabledTabs[0];
     }
-    return notDisabledTabs.find((tab, i) => i >= this.selectedIndex && tab !== currentTab);
+    // if dir is left, we filter for previous tabs instead
+    nextNotDisabledTabs = tabs.filter((tab, i) => !tab.disabled && i < this.selectedIndex);
+    return nextNotDisabledTabs[nextNotDisabledTabs.length - 1];
```

Comment:
I think the only issue with this approach is that we won't be able to cycle through tabs. If we're on the last not-disabled tab, pressing the arrow right key will not shift focus to the start of the tab list.